sdk/va-report-components/documentation/docs/api/ReportObjectResultData.md

```diff
@@ -0,0 +1,89 @@
+---
+id: ReportObjectResultData
+title: ReportObjectResultData
+---
+
+ReportObjectResultData objects contain the report object's data and additional metadata.
+
+## Properties
+
+### `resultName: string`
+
+The name of the associated query result. This name is needed for communicating any messages from a data-driven content
+object back to SAS Visual Analytics.
+
+### `rowCount: number`
+
+The number of rows of data returned. If all the data has been filtered out or no data items are assigned to the object,
+then the row count is 0.
+
+### `columns: Object[]`
+
+An array of column objects that let the author determine the type of data and other attributes such as format and label.
+Each column object supports the following properties:
+
+- `name: string` A unique name for the column.
+- `label: string` A human-readable label for the column.
+- `type: string` The data type of the entries in that column. Can be `'number'` or `'string'`.
+- `usage?: string` An optional string to describe how the column is used. The special value `'brush'` indicates that this column holds row
+  selection data. If a row is selected, then the value in the `'brush'` column has a value greater than `0`.
+- `aggregation?: string` An optional string that describes how the values in this column are aggregated together.
+- `format?: Object` An optional object describing how the data should be formatted for a column of type `'number'`. It has the
+  following properties:
+  - `name: string` A name for the format
```

Comment:
There's a period missing at the end of the description.

Reply:
Thanks Joy!